tasks/connectors/snyk_v2/readme.md

```diff
@@ -36,4 +37,5 @@ Complete list of Options:
 | kenna_connector_id | false | If set, we'll try to upload to this connector | n/a |
 | kenna_api_key | false | Kenna API Key | n/a |
 | kenna_api_host | false | Kenna API Hostname | api.kennasecurity.com |
+| Snyk_api_url | true | Snyk environment API base URL without prefix e.g. api.eu.snyk.io, api.snyk.io or api.au.snyk.io | n/a  |
```

Comment:
- parameter name in snyk_v2_task.rb is snyk_api_base
- parameters are case sensitive when running the toolkit image so we should use all lowercase in readme file to match our definitions

Reply:
Task updated with change, thanks!